plugins/organic/organic.cpp

```diff
@@ -233,8 +233,9 @@ void organicInstrument::playNote( NotePlayHandle * _n,
 	
 	if( _n->totalFramesPlayed() == 0 || _n->m_pluginData == NULL )
 	{
-		Oscillator * oscs_l[m_numOscillators];
-		Oscillator * oscs_r[m_numOscillators];
+		Oscillator * oscs_all = MM_ALLOC( Oscillator, m_numOscillators*2 );
```

Comment:
Seeing that `Oscillator` has overloaded `new` operators through the `MM_OPERATORS` macro, we should be able to replace this with

```cpp
Oscillator* oscs_all = new Oscillator[m_numOscillators * 2]
```